components/landing/TheQuickStart/StartLocally.vue

```diff
@@ -43,11 +43,11 @@
         >
           <cv-accordion-item>
             <template #title>
-              Building Qiskit requires a C++ compiler and development headers
+              Installing from source requires that you have the Rust compiler on your system
             </template>
             <template #content>
-              <lazy-prerequisites-for-linux v-if="selectedOs === OPERATING_SYSTEMS.linux" />
-              <lazy-prerequisites-for-mac v-else-if="selectedOs === OPERATING_SYSTEMS.mac" />
+              <lazy-prerequisites-for-linux-mac v-if="selectedOs === OPERATING_SYSTEMS.linux" />
+              <lazy-prerequisites-for-linux-mac v-else-if="selectedOs === OPERATING_SYSTEMS.mac" />
```

Comment:
```suggestion
              <lazy-prerequisites-for-linux-mac v-if="isLinuxOrMac" />
```

```
isLinuxOrMac = OPERATING_SYSTEMS.linux || OPERATING_SYSTEMS.mac
```

Reply:
Refactored in https://github.com/Qiskit/qiskit.org/pull/2904/commits/7c13de28f1e43a06e07c016c6b588cb144fcfa3d (please let me know if you agree to what I did - I had a tiny bit of struggle w/ the other approach but this is still a refactor)

Reply:
Could you extract `selectedOs === OPERATING_SYSTEMS.linux || selectedOs === OPERATING_SYSTEMS.mac` to a variable? that would be my last request :D

Reply:
This is the part that isn't working for me

For some reason, it never reaches the Windows condition..

(still investigating)

https://user-images.githubusercontent.com/6276074/214067190-ef1e41d0-c58f-446d-8172-e41c6c740ae6.mov